src/project.js

```diff
@@ -70,31 +70,51 @@ function projectionReducer(location) {
     newLocation.forestType = forestType;
   }
 
-  newLocation.altitudinalZone = getNextHeigtLevel(location.altitudinalZone);
+  if (location.altitudinalZone && targetaltitudinalZone !== undefined) {
+    newLocation.altitudinalZone = getNextHeigtLevel(location.altitudinalZone);
+  } else {
+    newLocation.altitudinalZone = location.altitudinalZone;
+  }
   return newLocation;
 }
 
-function project(location, targetaltitudinalZone) {
+function project(location = {}, targetaltitudinalZone) {
+  const altitudinalZonePointer = altitudinalZoneList.indexOf(
+    location.altitudinalZone,
+  );
+
   if (
+    targetaltitudinalZone &&
     types.altitudinalZone.find(v => v.code === targetaltitudinalZone) ===
-    undefined
+      undefined
   ) {
     throw new Error(
       `${targetaltitudinalZone} for target altitudinal zone is not valid.`,
     );
   }
 
-  const altitudinalZonePointer = altitudinalZoneList.indexOf(
-    location.altitudinalZone,
-  );
-
   let newLocation;
-  if (altitudinalZoneList[altitudinalZonePointer] !== targetaltitudinalZone) {
-    newLocation = projectionReducer(location);
-    if (newLocation.altitudinalZone !== targetaltitudinalZone) {
+
+  if (
+    altitudinalZoneList[altitudinalZonePointer] !== targetaltitudinalZone ||
+    targetaltitudinalZone === undefined
+  ) {
+    newLocation = projectionReducer(location, targetaltitudinalZone);
+    if (
+      newLocation.altitudinalZone !== undefined &&
```

Comment:
I guess we can remove line 104 because line 106 already checks if they are equal.

Reply:
otherwise the test breaks.